.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,11 +2,13 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: enhancement, help wanted
+labels: enhancement
 assignees: ''
 ---
 
-### ⚠️ BEFORE SUBMITTING, READ:
+<!--
+
+⚠️ BEFORE SUBMITTING, READ:
```

Comment:
why is this brought under comment?
```suggestion
<!--

⚠️ BEFORE SUBMITTING, READ:
```

Reply:
when you open a PR you would see it but if you forget to delete it with HTML comment it wont show later as it is not valid once you have opened an issue